config/routes.rb

```diff
@@ -19,7 +19,12 @@
 
   get 'workflows/:workflow_id/extractors/:extractor_id/extracts', to: 'extracts#index'
   put 'workflows/:workflow_id/extractors/:extractor_id/extracts', to: 'extracts#update'
+  get 'workflows/:workflow_id/extractors/export', to: 'data_requests#request_extracts'
 
   get 'workflows/:workflow_id/reducers/:reducer_id/reductions', to: 'reductions#index'
   put 'workflows/:workflow_id/reducers/:reducer_id/reductions', to: 'reductions#update'
+  get 'workflows/:workflow_id/reducers/export', to: 'data_requests#request_reductions'
+
+  get 'data_request/:request_id', to: 'data_requests#check_status'
```

Comment:
In retrospect that might have been wise, but at the time I wanted to put the endpoints for requesting the exports near the area where people would already be asking for json dumps (which is to say, in the workflows controller or the (hypothetical) users controller). it's not too late to change our minds about that, but i'd probably need to abandon this pull request to do so. your call :)